specs/vep-1553-downloads-badges-in-readmes/README.md

```diff
@@ -0,0 +1,170 @@
+
```

Comment:
@chrfoyer Thanks for this PR is looks great!
However I think you can remove this file completely. 
VEPs need to be created when we are making more architectural changes to the platform. Where ppl will have differing opinions.

Reply:
Removed with [this commit](https://github.com/vmware/versatile-data-kit/pull/2983/commits/6dd6a7d4bbbcfca7577fa56d9ebb041a58808d14). Thanks for the feedback!